wolfcrypt/src/cryptocb.c

```diff
@@ -141,6 +144,16 @@ static const char* GetHashTypeStr(int hash)
     return NULL;
 }
 
+#ifdef WOLFSSL_CMAC
+static const char* GetCmacTypeStr(int type)
+{
+    switch (type) {
+        case WC_CMAC_AES: return "AES";
```

Comment:
case without break or default might trigger analyzer warnings.

Reply:
Break shouldn't be necessary since this directly returns.  Concur that we could add defaults to all of these, but I'll follow the pattern for now.